src/mame/heathzenith/h89.cpp

```diff
@@ -996,7 +996,36 @@ ROM_START( h89_sigmasoft )
 	ROMX_LOAD("2716_mtrhex.u518",         0x0000, 0x0800, CRC(842a306a) SHA1(ddbc2b8bb127464af9eda8e7c56e6be7c8b43a16), ROM_BIOS(7))
 ROM_END
 
-static const auto rom_z89 = rom_h89;
+ROM_START( z89 )
+	ROM_REGION( 0x2000, "maincpu", ROMREGION_ERASEFF )
+	ROM_DEFAULT_BIOS("mtr90")
+
+	ROM_LOAD( "2716_444-19_h17.u520",     0x1800, 0x0800, CRC(26e80ae3) SHA1(0c0ee95d7cb1a760f924769e10c0db1678f2435c))
+
+	ROM_SYSTEM_BIOS(0, "mtr90", "MTR-90 (444-142)")
+	ROMX_LOAD("2732_444-142_mtr90.u518",  0x0000, 0x1000, CRC(c4ff47c5) SHA1(d6f3d71ff270a663003ec18a3ed1fa49f627123a), ROM_BIOS(0))
+
+	ROM_SYSTEM_BIOS(1, "mtr89", "MTR-89 (444-62)")
+	ROMX_LOAD("2716_444-62_mtr89.u518",   0x0000, 0x0800, CRC(8f507972) SHA1(ac6c6c1344ee4e09fb60d53c85c9b761217fe9dc), ROM_BIOS(1))
+
+	ROM_SYSTEM_BIOS(2, "mms84b", "MMS 444-84B")
+	ROMX_LOAD("2732_444_84b_mms.u518",    0x0000, 0x1000, CRC(7e75d6f4) SHA1(baf34e036388d1a191197e31f8a93209f04fc58b), ROM_BIOS(2))
+
+	ROM_SYSTEM_BIOS(3, "kmr-100", "Kres KMR-100 V3.a.02")
+	ROMX_LOAD("2732_kmr100_v3_a_02.u518", 0x0000, 0x1000, CRC(fd491592) SHA1(3d5803f95c38b237b07cd230353cd9ddc9858c13), ROM_BIOS(3))
+
+	ROM_SYSTEM_BIOS(4, "mtrhex_4k", "Ultimeth 4k ROM")
+	ROMX_LOAD("2732_mtrhex_4k.u518",      0x0000, 0x1000, CRC(e26b29a9) SHA1(ba13d6c9deef682a9a8262bc910d46b577929a13), ROM_BIOS(4))
+
+	ROM_SYSTEM_BIOS(5, "mtr90-84", "Heath's MTR-90 (444-84 - Superseded by 444-142)")
```

Comment:
Was this actually called "Heath's MTR-90"?  These systems are hard to find info on so I don't actually know, but it seems unlikely even for the 80s :-)

Reply:
I just looked at the manual, it shows "Zenith Data Systems MONITOR MTR-90" 
<img width="599" alt="Screenshot 2024-05-24 at 10 10 44 PM" src="https://github.com/mamedev/mame/assets/8291090/ece21d9a-b0af-4b39-a291-11e9304e2f55">

Reply:
My objection is to the "Heath's" part, because I assume it wouldn't be named like grape jelly.  If the ROM is just the Heath version of what is otherwise badged as a Zenith Data Systems MTR-90, I'd call that BIOS "MTR-90, Heathkit (444-84 - Superseded by 444-142)".

Reply:
The MTR-90 model numbers were actually the same between ZDS and Heath. I had just added the `Heath's` part, since the other ROMs had the company's name. Adding the `'s` wasn't intentional. I could just use `Zenith Data Systems` for the two MTR-90s, and `Heath` for the MTR-89 to make it consistent.

Reply:
Since there are questions on whether or not this PR will get merged, I'll make the description changes in a followup PR.

Reply:
Ok.  Given the 's wasn't intentional, I think this is OK if you fix that.